magento/magento2-login-as-customer
# magento/magento2-login-as-customer#168: Endless loading of the page after trying to search any word in "Login as Customer Log".

### Description (*)

Added fulltext search to the keyword search field. Currently searching only by "user_name" and "customer_email" fields.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2-login-as-customer#168: Endless loading of the page after trying to search any word in "Login as Customer Log".

### Manual testing scenarios (*)

1. See magento/magento2-login-as-customer#168: Endless loading of the page after trying to search any word in "Login as Customer Log".

### Questions or comments


### Contribution checklist (*)
